bin/cloudgov/create-service-account

```diff
@@ -1,18 +1,31 @@
-#!/bin/bash
+#!/bin/env bash
```

Comment:
Should be /usr/bin/env (I don't have /bin/env, and we already use /usr/bin/env in many places)